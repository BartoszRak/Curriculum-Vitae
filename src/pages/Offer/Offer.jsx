import React from 'react'
import PropTypes from 'prop-types'

import useStyle from './Offer.style'

export function Offer({ classes: overridingClasses }) {
  const classes = { ...useStyle(), ...overridingClasses }
  return <div>offer</div>
}

Offer.propTypes = {
  classes: PropTypes.object,
}

export default Offer
