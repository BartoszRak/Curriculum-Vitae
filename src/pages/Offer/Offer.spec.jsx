import React from 'react'
import { cleanup } from '@testing-library/react'
import { shallow } from 'enzyme'

import { Offer } from './Offer'

jest.unmock('@material-ui/core')

describe('component: Offer', () => {
  afterEach(cleanup)

  describe('rendering', () => {
    test('render without crash', () => {
      const wrapper = shallow(<Offer />)

      expect(wrapper).toBeTruthy()
    })

    test('match snapshot', () => {
      const wrapper = shallow(<Offer />)

      expect(wrapper).toMatchSnapshot()
    })
  })
})
