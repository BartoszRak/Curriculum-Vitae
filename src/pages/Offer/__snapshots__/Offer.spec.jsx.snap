// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component: Offer rendering match snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  offer
</div>
`;
